src/Resources/contao/library/Contao/DcaLoader.php

```diff
@@ -85,7 +85,10 @@ public function load($blnNoCache=false)
 					include $file;
 				}
 			}
-			catch (\InvalidArgumentException $e) {}
+			catch (\InvalidArgumentException $e)
+			{
+				\System::getContainer()->get('monolog.logger.contao')->notice($e->getMessage(), ['exception' => $e]);
```

Comment:
> Maybe we should create a PR for Symfony to throw a `ResourceNotFoundException` in the next version?

The Symfony `FileLocator` class throws the `\InvalidArgumentException`, therefore we cannot change this (except by replacing the class). But the following is wrong anyway:

```php
class PaletteNotFoundException extends \InvalidArgumentException
```

All palette errors are `\LogicException`s and if the palette exceptions extended these, they would not be caught by the file locator routine.